tests/installation/releasenotes.pm

```diff
@@ -56,11 +56,16 @@ sub run() {
 
     # no relnotes for ltss in QAM_MINIMAL
     push @no_relnotes, qw(ltss) if get_var('QAM_MINIMAL');
+    # no HA-GEO release-notes for SLE12-SP1 on s390x
+    if (check_var('ARCH', 's390x') and check_var('VERSION', '12-SP1')) {
```

Comment:
I guess you need to use BASE_VERSION here, because VERSION for upgrades is still 12-SP3